infra/base-images/base-builder/compile_afl

```diff
@@ -43,4 +43,17 @@ popd > /dev/null
 export CC="$SRC/aflplusplus/afl-clang-fast"
 export CXX="$SRC/aflplusplus/afl-clang-fast++"
 
+# Set sane afl++ environment defaults:
+# Be quiet, otherwise this can break some builds
+export AFL_QUIET=1
+# Several targets run their own tools, so ensure its working
+export AFL_MAP_SIZE=4194304
+# Disable sanitizers during builds
```

Comment:
This is not good to go as bad build failure will show as like segfault instead of proper sanitizer stacks (and otherwise painful for debugging.). If leaks is the only expected thing, can do
export ASAN_OPTIONS="$ASAN_OPTIONS:detect_leaks=0"

Reply:
there was also one UBSAN issue I came across, but that would also be an issue for runs.